tutorial-3.2/README.md

```diff
@@ -4,6 +4,15 @@ This tutorial is meant to demonstrate how to set up, run, and analyze a membrane
 ## Tutorial files
 
 All files necessary for completing the tutorial can be found in this directory.
+Please install required packages before running the tutorial by running the following command:
+```
+conda env create --file=environment.yaml
```

Comment:
The file is environment.yml (without the a)

Reply:
Nice catch. Fixed.